emissionsapi/web.py

```diff
@@ -64,6 +64,16 @@ def decorated(*args, **kwargs):
         country = kwargs.pop('country', None)
         polygon = kwargs.pop('polygon', None)
         point = kwargs.pop('point', None)
+
+        number_of_parameter = sum(x is not None for x in (
+            geoframe, country, polygon, point))
+        if number_of_parameter == 0:
```

Comment:
Do you have to specify one of these parameters? I sometimes don't if I want to get data for a narrow time-span and I don't really care where the measurement is located. Though, admittedly, that's usually for debugging or testing purposes.

Reply:
Making one of those query parameter mandatory makes it less easy to screw things accidentally up by asking for too much data.

What time span do you usually use for those tests? I tested this

```bash
curl -X GET "https://demo.emissions-api.org/api/v1/geo.json?begin=2019-02-10&end=2019-02-11" -H  "accept: application/json"
```
with our current server setup and got a timeout after 31 seconds :smile:

I think the main question is, if there is a reasonable use case for not having one of those parameters set. Debugging and Testing excluded.